testsuite/tests/singlecluster/identical_hostnames/auth/conftest.py

```diff
@@ -0,0 +1,18 @@
+"""Conftest for "identical hostname" tests"""
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization):
+    """1st 'allow-all' Authorization object"""
+    authorization.authorization.add_opa_policy("rego", "allow = true")
+    return authorization
+
+
+@pytest.fixture(scope="module")
+def rate_limit():
```

Comment:
Common practice across testsuite is creating new `commit` fixtures, with objects you want to create. In this case just `authorization`

Reply:
So in theory this can be removed I assume. I have a `commit` fixture defined in the test files directly and RLP is not committed there.

Reply:
It seems it indeed can be removed. Will force push and let you know.

Reply:
@jsmolar force-pushed now